packages/cli/src/commands/org/config/preview.ts

```diff
@@ -93,7 +75,49 @@ export default class Preview extends Command {
     return cfg.organization;
   }
 
+  private async handleInvalidSnapshot(snapshot: Snapshot) {
+    // TODO: CDX-362: handle invalid snashot cases
```

Comment:
can be ignored, 50% suggestion, 50% trolling :P 
```suggestion
    // TODO: CDX-362: handle invalid snapshot cases
```